Shaders/PBR/private/ComputeIrradianceMap.psh

```diff
@@ -57,18 +57,19 @@ float3 IrradianceMap(float3 N)
         // Solid angle of current smple
         float OmegaS = 1.0 / (float(g_NumSamples) * pdf);
         
-        float NumPixels = g_EnvMapWidth * g_EnvMapHeight;
+        // Solid angle of 1 pixel across all cube faces
 #if ENV_MAP_TYPE == ENV_MAP_TYPE_CUBE
-        NumPixels *= 6.0;
+        float OmegaP = 4.0 * PI / (6.0 * g_EnvMapWidth * g_EnvMapHeight);
```

Comment:
I would suggest adding two functions to PBR_Common.fxh for clarity and reducing copy-paste:
`GetCubeMapPixelAngularSize` and `GetSphereMapPixelAngularSize`